### DISCUSSION TITLE
Provide github action to automatically update vcpkg port file on release

### DISCUSSION THREAD

**Discussion by jwillemsen** [microsoft/vcpkg#17834] [Ideas]
****

**Is your feature request related to a problem? Please describe.**
For projects hosted on github it would be convenient to have a vcpkg action to automatically update the port in vcpkg when there is a new release

**Proposed solution**
Easy to use vcpkg github action to automatically update the port and create a PR on a release of a project on github

**Describe alternatives you've considered**
Manually update port file and make a PR

**Comment by johnwason** [comment]
I have done some partial work on an action like this for a private registry:

https://github.com/robotraconteur/vcpkg-robotraconteur/blob/master/.github/workflows/update_port.yml

https://github.com/robotraconteur/vcpkg-robotraconteur/blob/master/scripts/update_port.py

This does not call `vcpkg` to update the version database.
